Documentation/faq.md

```diff
@@ -10,6 +16,24 @@
 
 ### Deployment
 
+#### System requirements
+
+Since etcd writes data to disk, SSD is highly recommended. Default storage size limit is 2GB, configurable up to 8GB. And allocate as much heap size to avoid disk swap. At CoreOS, etcd cluster is usually deployed on dedicated CoreOS Container Linux machines, with dual-core processors, 2GB of RAM, and 80GB of SSD *at the very least*. **It highly varies on workloads; please test before production deployment**.
```

Comment:
2GB is the 32-bit mmap max map size in boltdb, and also reasonable limit for our use case.
8GB is also arbitrary, but we want this to prevent degrading performance. See mvcc for more details.
RAM should be at least 2GB for etcd or bigger for quota size. This is the RAM recommendation.